internal/pkg/model/receiver.go

```diff
@@ -33,3 +34,169 @@ type Receiver struct {
 	Name      string `json:"name" validate:"required,min=1,max=40"`
 	Secret    string `json:"secret" validate:"required,len=48"`
 }
+
+// To add a new column type `Foo`:
+// - create a struct for it: `type ColumnFoo struct{}`
+// - create a type name constant for it: `columnIDFoo = "foo"`
+// - add it to `columnToType` and `typeToColumn`
+// - implement `MappedColumn` for `ColumnFoo`
+
+type MappedColumns []MappedColumn
```

Comment:
Je to naimplementované podle https://goplay.tools/snippet/4Sr_LspTuX-

Jediný rozdíl je v tom, že jsem tady přidal "dummy interface" `MappedColumn`, aby se tady nemuselo zacházet čistě s `any` a aby to bylo trochu lepší na používání, protože do `MappedColumns` nepůjde dosadit nic než ty column types definovaný tady.

Reply:
👍 Akorát si říkám, že ty `Column*` struktury by se měly oprefixovat nebo přesunout do nějakýho sub-package. Aby bylo jasný, že jde o mapping sloupců a nepletlo se to s definicí Storage tabulek jakmile začneme do CLI přidávat jejich podporu. (Proto jsem už ty sloupce pojmenoval `MappedColumns`.) @michaljurecko Co myslíš?

Reply:
:slightly_smiling_face: akurat som to paralelne s tebou takto okomentoval
https://github.com/keboola/keboola-as-code/pull/997#discussion_r1028012695